molior/molior/server.py

```diff
@@ -84,6 +85,21 @@ def destroy_cirrina_context(cirrina):
     cirrina.db_session.close()
 
 
+class MoliorAccessLogger(AbstractAccessLogger):
+
+    def _get_username(self, request):
+        try:
+            return request.cirrina.web_session.get('username')
+        except Exception:
+            return None
```

Comment:
Maybe log what the error was?

Reply:
I am not sure we want to log within a log handler. I wanted to explicitly avoid having to think about what would happen if by calling log() from within log handlers. I guess we could `print` instead, but then one looses all the handling that logs may be configured to do (e.g. logging to a database, or syslog, etc.)

Reply:
Fair :)